app/src/main/java/nl/tudelft/bbw/controller/TrustController.java

```diff
@@ -63,13 +63,13 @@ public static Block verifiedIBAN(Block block) {
     }
 
     /**
-     * revokeBlock
+     * revokeBlockTrust
      * Sets the block value to the revoked trust value
      *
      * @param block given block to revoke
      * @return block with the new trust value
      */
-    public static Block revokeBlock(Block block) {
+    public static Block revokeBlockTrust(Block block) {
```

Comment:
All the API names will be renamed after the analysis process of the Interaction Design interviews is finished. Hang on.